src/test/java/io/vertx/test/core/VertxOptionsTest.java

```
@@ -416,4 +424,15 @@ public void testJsonOptions() {
     assertEquals(warningExceptionTimeUnit, options.getWarningExceptionTimeUnit());
     assertEquals(blockedThreadCheckIntervalUnit, options.getBlockedThreadCheckIntervalUnit());
   }
+
+  @Test
+  public void testNullFileSystemOptions() {
+    VertxOptions options = new VertxOptions().setFileSystemOptions(null);
+    try {
+      options.isFileResolverCachingEnabled();
+      options.setFileResolverCachingEnabled(true);
+    } catch (NullPointerException e) {
+      fail("Should handle null FileSystemOptions");
```

just let it go through and let the NPE fail the test